packages/flutter_adaptive_scaffold/lib/src/breakpoints.dart

```diff
@@ -213,9 +246,10 @@ class Breakpoint {
 
     final bool isHeightActive = isDesktop ||
         orientation == Orientation.portrait ||
-        (orientation == Orientation.landscape &&
-            height >= lowerBoundHeight &&
-            height < upperBoundHeight);
+        (orientation == Orientation.landscape && andUp
```

Comment:
Should we change the documentation for `andUp` to reflect that it now includes any height above the set height?

I wonder if `isWidthActive` should we just defer to it? i.e change it to below or am I thinking about this wrong.

```dart
    final bool isHeightActive = isDesktop ||
        orientation == Orientation.portrait ||
        (orientation == Orientation.landscape && andUp
            ? isWidthActive || height >= lowerBoundHeight
            : height >= lowerBoundHeight && height < upperBoundHeight);
```

Reply:
You are right. Thanks!